### ISSUE TITLE
Raise warning if sex prediction is based on too few exons

### ISSUE THREAD

**Issue by robinandeer** [robinandeer/chanjo#152]
Set a reasonable cutoff for how many exons the prediction is based on (X/Y)

**Comment by biocyberman** [comment]
Obviously, low coverage makes sex outcome uncertain (you can read that again). I tried getting sex for some, and they result in all male. 

Ahem ...!
So the coverage ration between female(X) and male (Y) should also be considered. Because male (Y) is usually covered less than female (X). The outcome of X over Y (X/Y) is greater than Y over X (Y/X). 

Just some wild thoughts into difficult but intriguing problem.
